packages/mobile/android/app/src/main/java/org/celo/mobile/MainApplication.java

```diff
@@ -60,7 +59,7 @@ public ReactNativeHost getReactNativeHost() {
   public void onCreate() {
     super.onCreate();
     SoLoader.init(this, /* native exopackage */false);
-    initializeFlipper(this); // Remove this line if you don't want Flipper enabled
+    initializeFlipper(this, getReactNativeHost().getReactInstanceManager());
```

Comment:
Should this be disabled somehow when app is built for release?

Reply:
This is handled below in the initializeFlipper method - if the class isn't found, the call fails